corehq/apps/userreports/tasks.py

```diff
@@ -35,7 +35,6 @@ def rebuild_indicators(indicator_config_id):
             adapter.save(doc)
         except DataError as e:
             logging.exception('problem saving document {} to table. {}'.format(doc['_id'], e))
-    adapter.engine.dispose()
```

Comment:
@snopoke thinking about it a bit more I think we don't want to dispose it. the engine can be a "global" defined in the `connection_manager` similar to how it's defined at the module level in your link. since many tasks might share the engine I think we shouldn't dispose it and just use that one. I do think we need to make sure we close connections, but my understanding is that that is handled by using the `with engine.begin()` structure inside the adapter class. am i missing something?